Malt/Shaders/Common/Matrix.glsl

```diff
@@ -2,22 +2,24 @@
 #define COMMON_MATRIX_GLSL
 
 /* META GLOBAL
-    @meta: category=Math; subcategory=Mat4;
+    @meta: category=Math; subcategory=Matrix Math;
```

Comment:
whats the reasoning behind naming this subcategory 'Matrix Math'? the math part would be redundant and inconsistent with the rest of the subcategories

Reply:
I've renamed (some of) the other subcategories too.
In the case of Matrix there's no overlap (yet?), but in the other cases is to prevent the parameter and the math node from having the same name.

Would you prefer another alternative?